src/pages/AddBoxForm/AddBoxForm.css

```diff
@@ -1,5 +1,5 @@
 .add-box-form-title {
   font-size: 2rem;
-  font-weight: 500;
-  margin: 30px 0 0 50px;
+  font-weight: 600;
+  margin: 30px 0 10px 365px;
```

Comment:
Could we change this to use % instead of px? This will ensure the styling looks good on different screen sizes. Please try to do this for other styling in this PR as well!